envoy_build_config/extensions_build_config.bzl

```diff
@@ -133,22 +141,37 @@ EXTENSIONS = {
     "envoy.tracers.datadog":                            "@envoy//source/extensions/tracers/datadog:config",
     "envoy.tracers.zipkin":                             "@envoy//source/extensions/tracers/zipkin:config",
     "envoy.tracers.opencensus":                         "@envoy//source/extensions/tracers/opencensus:config",
+    # WiP
```

Comment:
Are we missing something here?

Reply:
It's from the original file from Envoy so no.